ts/utils/paymentMethod.ts

```diff
@@ -228,3 +230,17 @@ export const isBancomatBlocked = (pan: Card) =>
 
 export const isCoBadgeOrPrivativeBlocked = (pan: PaymentInstrument) =>
   pan.validityStatus === ValidityStatusEnum.BLOCK_REVERSIBLE;
+
+export const isPaymentMethodExpired = (
+  pm: RawPaymentMethod
+): Either<Error, boolean> => {
+  switch (pm.kind) {
+    case "BPay":
+    case "Satispay":
+      return left(Error(`${pm.kind} has no expiring date`));
```

Comment:
I was thinking if it shouldn't be `right(false)` for a payment method that never expires, what you think?

Reply:
Since it is not an error, I'm agree that these method never expire
Changed in https://github.com/pagopa/io-app/pull/2982/commits/1e8d0910a2bd4132bbb355e6808be940e2d67854